request-reply.md

```diff
@@ -19,8 +19,11 @@ To successfully trigger an invocation, an http request MUST adhere to the follow
 An invoker MUST only consider POST requests on `/` for invocations. Other methods or other paths MUST NOT lead to a function invocation. In those cases, an invoker MAY respond with an error condition or an unrelated successful payload.
 
 An incoming http request SHOULD contain a `Content-Type` header that will be used by the invoker to drive deserialization of the http request *body*. If the http request does not bear such a header, an invoker MUST interpret the body as having an `application/octet-stream` content-type.
+If the incoming http request `Content-Type` header value denotes a textual content type, the header MAY contain a charset attribute, such as: `text/csv;charset=ISO-2022-CN`. In absence of such an attribute for a textual content type, the default charset UTF-8 will be assumed.
 
 An incoming http request SHOULD contain an `Accept` header that will be used by the invoker to drive serialization of the invocation result back to the client. In the absence of such a header, an invoker MUST interpret the client as accepting an `*/*` response.
+If the `Accept` header denotes textual MIME types, their corresponding charsets MAY be specified via the `Accept-Charset` header.
+In absence of such a header, every textual MIME type will be assumed to be encoded with charset UTF-8.
```

Comment:
Don't think so. In the absence of accept information, we should assume `*`, ie "give me whatever you want"